### PR TITLE
docs: Fix anchor in URL

### PR DESCRIPTION
**Pull request by karlhorky**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[X] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)

Fix the anchor in the link

### CODE REVIEW

**Review by kecrily — APPROVED** [review]
LGTM, thanks

File: docs/src/user-guide/configuring/configuration-files.md [hunk 1]
```diff
@@ -424,7 +424,7 @@ If a config is provided via the `--config` CLI option, the glob patterns in the
 
 If you specified directories with CLI (e.g., `eslint lib`), ESLint searches target files in the directory to lint. The target files are `*.js` or the files that match any of `overrides` entries (but exclude entries that are any of `files` end with `*`).
 
-If you specified the [`--ext`](https://eslint.org/docs/user-guide/command-line-interface#ext) command line option along with directories, the target files are only the files that have specified file extensions regardless of `overrides` entries.
+If you specified the [`--ext`](https://eslint.org/docs/user-guide/command-line-interface#--ext) command line option along with directories, the target files are only the files that have specified file extensions regardless of `overrides` entries.
```
Comment (amareshsm) [inline docs/src/user-guide/configuring/configuration-files.md:427]:
```suggestion
If you specified the [`--ext`](../command-line-interface#--ext) command line option along with directories, the target files are only the files that have specified file extensions regardless of `overrides` entries.
```

Comment (karlhorky) [reply docs/src/user-guide/configuring/configuration-files.md:427]: I'm fine with making this change, if @kecrily prefers that we add this too.

Comment (karlhorky) [reply docs/src/user-guide/configuring/configuration-files.md:427]: Or either of you can commit this - also no problem

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks!

**Comment by karlhorky** [comment]
Glad to help, thanks for the merge!

**Pull request merged**